content/en/docs/concepts/signals/traces.md

```diff
@@ -289,17 +289,28 @@ another.
 
 ### Span Status
 
-A status will be attached to a span. Typically, you will set a span status when
-there is a known error in the application code, such as an exception. A Span
-Status will be tagged as one of the following values:
+Each span has a status. The three possible kinds are:
 
 - `Unset`
-- `Ok`
 - `Error`
+- `Ok`
+
+The default value is `Unset`. A span status that is `Unset` means that the
+operation it tracked successfully completed without an error.
+
+When a span status is `Error`, then that means some error ocurred. For example,
+this could be due to an HTTP 500 error on a server.
+
+When a span status is `Ok`, then that means the span was explicitly marked as
+error-free, even if an error may have actually occurred. For example, an
+instrumentation library may decide to set a span status to `Error`, but that may
+actually be an incorrect status to assign in your application's specific
+context. In such a case, you can change the status to `Ok`, causing an
+Observability backend to not count this span as an error.
```

Comment:
So in general I agree that the spec made the wrong design here. And moreover, I think there might need to be better guidance for users of Observability backends around how you query this kind of data. In generally, "show me what wasn't an error" is what you want, but some UIs (like Tempo) let you filter specifically by a span status and that's where some confusion arises. If the spec detailed things in a more user-friendly way then this wouldn't be an issue, but alas I don't think it can be changed now.

> The primary use case is the application owner with domain knowledge explicitly sets this status to OK

I'd agree, but I'd love to have a more specific example though. When there's no real difference between an actually-complete-and-no-error span set to OK and left Unset, then I'm not sure what the benefit to explicitly setting it to OK is.

Reply:
There was an odd critique in the OTEP about the approach that OpenTracing had, where we had `error=true` attribute. I actually think it was pretty good, much better than this confusion between Unset and OK. Since that status was elevated to the data model (instead of semconv in OT), I would prefer the status to be OK or Error and not have Unset as an option. Unset is really an intermediate state that could be managed in the SDK until the span is finished. Once it's finished, if the error status was not set then the span is automatically OK. Since OTEL already has Unset in a stable spec, I think the path forward is to tweak the spec to say that Tracer should change Unset->OK when the span is finished. The other train of thought in the OTEP that some _backend_ components might change Unset to OK is completely alien to me - why would a backend have any better knowledge about what happened in the application than the application & instrumentation that produced the data?

Reply:
I think this is a valuable discussion, can we turn this into a spec issue? For the docs I think we need to document the _as is_ state and then later update it accordingly.

Reply:
I think in the interim we should tweak the example(s) in prose a bit, more to this point:

> I don't like this emphasis or overriding error, this is not the primary use case for OK status.

Reply:
@yurishkuro I adjusted the wording here and in the preamble. PTAL?